src/algebra/hom/equiv.lean

```diff
@@ -255,13 +255,13 @@ theorem self_comp_symm (e : M ≃* N) : e ∘ e.symm = id := funext e.apply_symm
 @[simp, to_additive]
 theorem coe_refl : ⇑(refl M) = id := rfl
 
-@[to_additive]
+@[simp, to_additive]
 theorem refl_apply (m : M) : refl M m = m := rfl
```

Comment:
Won't the simp_nf linter complian about the coe_refl lemma above? I guess we'll find out.

Reply:
Seems to be happy.

Reply:
Seems to be happy.